src/java.base/share/classes/javax/net/ssl/SSLSessionContext.java

```diff
@@ -88,8 +88,9 @@ public interface SSLSessionContext {
      * If the timeout limit is set to 't' seconds, a session exceeds the
      * timeout limit 't' seconds after its creation time.
      * When the timeout limit is exceeded for a session, the
-     * <code>SSLSession</code> object is invalidated and future connections
-     * cannot resume or rejoin the session.
+     * <code>SSLSession</code> object is marked so that future connections
+     * cannot resume or rejoin the session. Sessions can continue to be used
```

Comment:
Maybe make it more clear these are active sessions -  "Active sessions can continue to ..."

Reply:
Good idea - changed.